### PR TITLE
Update UG for Lesson Add

### PR DESCRIPTION
**Pull request by eeliana**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -282,24 +282,25 @@ Adds a lesson with the corresponding details to the specified student in TAB.
...
 Examples:
 
-* `list` followed by `ladd 1 recurring/ date/16 Sep 2021 start/15:00 end/16:00 subject/Math` (recurring lesson)
-adds the lesson to the 1st student in TAB.
+* `list` followed by `ladd 1 recurring/ date/16 Sep 2021 time/1500-1600 subject/Math`
+  adds the recurring lesson to the 1st student in TAB.
 
-* `find john` followed by `ladd 1 date/16 Sep 2021 start/15:30 end/17:30 subject/Science hw/TYS p2 Q2` 
-  (makeup lesson w/ homework) adds the lesson to the 1st student in the results of the `find` command.
+* `find john` followed by `ladd 1 date/16 Sep 2021 start/15:30 end/17:30 subject/Science hw/TYS p2 Q2`
```
Comment (eeliana) [inline docs/UserGuide.md:302]:
```suggestion
* `find john` followed by `ladd 1 date/16 Sep 2021 time/1530-1730 subject/Science hw/TYS p2 Q2`
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -282,24 +282,25 @@ Adds a lesson with the corresponding details to the specified student in TAB.
...
 Examples:
 
-* `list` followed by `ladd 1 recurring/ date/16 Sep 2021 start/15:00 end/16:00 subject/Math` (recurring lesson)
-adds the lesson to the 1st student in TAB.
+* `list` followed by `ladd 1 recurring/ date/16 Sep 2021 time/1500-1600 subject/Math`
+  adds the recurring lesson to the 1st student in TAB.
 
-* `find john` followed by `ladd 1 date/16 Sep 2021 start/15:30 end/17:30 subject/Science hw/TYS p2 Q2` 
-  (makeup lesson w/ homework) adds the lesson to the 1st student in the results of the `find` command.
+* `find john` followed by `ladd 1 date/16 Sep 2021 time/1530-1730 subject/Science hw/TYS p2 Q2`
```
Comment (lingshanng) [inline docs/UserGuide.md:302]: can add multiple hw, and vary the other fields a bit

File: docs/UserGuide.md [hunk 3]
```diff
@@ -269,24 +269,25 @@ Adds a lesson with the corresponding details to the specified student in TAB.
...
 Examples:
 
-* `list` followed by `ladd 1 recurring/ date/16 Sep 2021 start/15:00 end/16:00 subject/Math` (recurring lesson)
-adds the lesson to the 1st student in TAB.
+* `list` followed by `ladd 1 recurring/ date/30 jan 2022 time/0900-1100 subject/Math`
+  adds the recurring lesson to the 1st student in TAB.
 
-* `find john` followed by `ladd 1 date/16 Sep 2021 start/15:30 end/17:30 subject/Science hw/TYS p2 Q2` 
-  (makeup lesson w/ homework) adds the lesson to the 1st student in the results of the `find` command.
+* `find john` followed by `ladd 1 date/16 Sep 2021 time/1530-1730 subject/Science hw/TYS p2 Q2 hw/Exercise 3 hw/Lab report`
```
Comment (lingshanng) [inline docs/UserGuide.md:289]:
```suggestion
* `find n/john` followed by `ladd 1 date/16 Sep 2021 time/1530-1730 subject/Science hw/TYS p2 Q2 hw/Exercise 3 hw/Lab report`
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -269,24 +269,25 @@ Adds a lesson with the corresponding details to the specified student in TAB.
...
-* Start time `HH:mm`
-* End time `HH:mm`
+* Time range `HHmm-HHmm`
 * Subject
 * Homework
 
-Format: `ladd INDEX [recurring/] date/dd MMM yyyy start/HH:mm end/HH:mm subject/SUBJECT [hw/HOMEWORK]`
+Format: `ladd INDEX [recurring/] date/dd MMM yyyy time/HHmm-HHmm subject/SUBJECT [hw/HOMEWORK]…​`
+
+* The date is case-insensitive. i.e. `12 jaN 2022` is equivalent to `12 JAN 2022`.
```
Comment (lingshanng) [inline docs/UserGuide.md:278]: i think you can have a "Notes about lesson add command" block and merge with the tip below.
maybe lesson delete also can put inside a block

File: docs/UserGuide.md [hunk 5]
```diff
@@ -269,39 +269,57 @@ Adds a lesson with the corresponding details to the specified student in TAB.
...
 * Homework
 
-Format: `ladd INDEX [recurring/] date/dd MMM yyyy start/HH:mm end/HH:mm subject/SUBJECT [hw/HOMEWORK]`
+Format: `ladd INDEX [recurring/] date/dd MMM yyyy time/HHmm-HHmm subject/SUBJECT [hw/HOMEWORK]…​`
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about the `ladd` command:**<br>
+
+* The type of lesson will be inferred from the presence of the "recurring/" prefix.
```
Comment (Chesterwongz) [inline docs/UserGuide.md:282]: I don't think we used double quotations anywhere else? is it ok to use it here?

**Review by lingshanng — APPROVED** [review]
LGTM

**Pull request merged**